vendor/golang.org/x/net/websocket/websocket.go

```diff
@@ -5,11 +5,11 @@
 // Package websocket implements a client and server for the WebSocket protocol
 // as specified in RFC 6455.
 //
-// This package currently lacks some features found in an alternative
-// and more actively maintained WebSocket package:
+// This package currently lacks some features found in alternative
+// and more actively maintained WebSocket packages:
 //
 //     https://godoc.org/github.com/gorilla/websocket
-//
+//     https://godoc.org/nhooyr.io/websocket
```

Comment:
I noticed this change in the comment, which relates to https://github.com/golang/go/issues/18152

It's used in k8s.io/apiserver/pkg/util/wsstream/stream.go, so perhaps that should be reviewed if the alternatives are better for how it's used.